scripts/variantstore_wdl/bq_ingest_arrays.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+set -e
+
+if [ $# -lt 5 ]; then
+  echo "usage: $0 <project-id> <dataset-name> <storage-location> <table-id> <load> <uuid>"
+  exit 1
+fi
+
+PROJECT_ID=$1
+DATASET_NAME=$2
+STORAGE_LOCATION=$3
+TABLE_ID=$4
+if [ $5 == "true" ]; then
+  LOAD=true
+else
+  LOAD=false
+fi
```

Comment:
when do you not want to load?

Reply:
My thought was that if we end up using Google Data Transfer we need a script that will create the tables but not actually load the data. Ideally we'd add the generation of the Transfers to this script too, but I didn't get around to doing that.

Now that I look at this though, I think the code for this script is all contained within the WDL and I shouldn't have committed this extra file. I'll add a comment about the Google Data Transfer to the WDL and delete this bash script.